apps/src/templates/referenceGuides/ReferenceGuideView.jsx

```diff
@@ -1,21 +1,89 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
+import NavigationBar from './NavigationBar';
+import color from '@cdo/apps/util/color';
+import {Link} from '@dsco_/link';
+import {organizeReferenceGuides} from '@cdo/apps/util/referenceGuideHelpers';
+import classNames from 'classnames';
 
-export default function ReferenceGuideView({referenceGuide}) {
-  return (
-    <div>
-      <h1>{referenceGuide.display_name}</h1>
-      <EnhancedSafeMarkdown markdown={referenceGuide.content} />
-    </div>
-  );
-}
+const baseUrl = window.location.href
+  .split('/')
+  .slice(0, -1)
+  .join('/');
 
 const referenceGuideShape = PropTypes.shape({
   display_name: PropTypes.string,
-  content: PropTypes.string
+  content: PropTypes.string,
+  position: PropTypes.number,
+  parent_reference_guide_key: PropTypes.string
 });
 
+const NavBarItem = ({guide, isActive}) => (
```

Comment:
I think it would be good for NavBarItem to be in a separate file because it is going to be used by anything that uses the nav bar such as code docs